### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Jamie-BitFlight**

### CODE REVIEW

**Review by sarkis — COMMENTED** [review]
This looks 👍. We're moving all of our README.md to being generated by README.yaml. See: 
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/README.yaml
https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/Makefile

Once you have those in place it is just:
bootstrap:
```
make init
make readme/deps
```
generate README.md from README.yaml
```
make readme
```

**Comment by sarkis** [comment]
Also don't forget to commit up `docs/` and add `.build-harness` and `build-harness` to the .gitignore

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+#   - "docs/variables.md"
+#   - "docs/outputs.md"
+
+# Contributors to this project
+contributors:
+  - name: "Erik Osterman"
+    homepage: "https://github.com/osterman"
+    avatar: "http://s.gravatar.com/avatar/88c480d4f73b813904e00a5695a454cb?s=144"
+    github: "osterman"
+  - name: "Jamie Nelson"
```
Comment (osterman) [inline README.yaml:149]: Whoot! =)

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+# Example usage
+#examples: |-
+#  Example goes here...
+
+# How to get started quickly
+#quickstart: |-
+#  Here's how to get started...
+
+# Other files to include in this README from the project folder
+# include:
```
Comment (osterman) [inline README.yaml:139]: Add this so it's included in the `README.md`

```
include:
  - "docs/terraform.md"
  - "docs/targets.md"
```

File: README.yaml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  }
+
+  resource "aws_iam_policy" "ps_manage" {
+    name_prefix   = "manage_any_parameter_store_value"
+    path   = "/"
+    policy = "${module.ps_policy.manage_parameter_store_policy}"
+  }
+  ```
+
+# Example usage
```
Comment (osterman) [inline README.yaml:130]: We can drop this now that you added it.

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
+variable "parameter_root_name" {
+  description = "The prefix or root parameter that you want to allow access to"
```
Comment (sarkis) [inline variables.tf:2]: For consistency use punctuation `.` to end sentence here

**Review by sarkis — APPROVED** [review]
minor nitpick in inline comment

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (osterman) [inline LICENSE:189]: Please update copyright =)

File: README.yaml [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+    url: "https://github.com/cloudposse/terraform-aws-ssm-parameter-store-policy-documents/releases"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-aws-ssm-parameter-store"
+    description: "AWS SSM Parameter Store module"
+    url: "https://github.com/cloudposse/terraform-aws-ssm-parameter-store"
+
```
Comment (osterman) [inline README.yaml:35]: Add:

```
 - name: "terraform-aws-ssm-iam-role"
   description: "Terraform module to provision an IAM role with configurable permissions to access SSM Parameter Store"   
   url: https://github.com/cloudposse/terraform-aws-ssm-iam-role
 - name: "terraform-aws-kms-key"
   description: "Terraform module to provision a KMS key with alias"
   url: https://github.com/cloudposse/terraform-aws-kms-key
```

File: README.yaml [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+
+# Short description of this project
+description: |-
+  This module generates JSON documents for restricted permission sets for AWS SSM Parameter Store access.
+  Helpful when combined with [terraform-aws-ssm-parameter-store](https://github.com/cloudposse/terraform-aws-ssm-parameter-store)
+# How to use this project
+
```
Comment (osterman) [inline README.yaml:43]: Drop whitespace

File: README.yaml [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+  - name: "terraform-aws-ssm-parameter-store"
+    description: "AWS SSM Parameter Store module"
+    url: "https://github.com/cloudposse/terraform-aws-ssm-parameter-store"
+
+
+
+# Short description of this project
+description: |-
+  This module generates JSON documents for restricted permission sets for AWS SSM Parameter Store access.
+  Helpful when combined with [terraform-aws-ssm-parameter-store](https://github.com/cloudposse/terraform-aws-ssm-parameter-store)
```
Comment (osterman) [inline README.yaml:41]: Add new line after...

File: outputs.tf [hunk 9]
```diff
@@ -0,0 +1,24 @@
+output "read_parameter_store_policy" {
+  description = "A JSON policy document that only allows read access to the parameter store"
+  value = "${data.aws_iam_policy_document.read_parameter_store.json}"
+}
+
+output "write_parameter_store_policy" {
+  description = "A JSON policy document that only allows write access to the parameter store "
```
Comment (osterman) [inline outputs.tf:7]: trailing whitespace

**Review by osterman — APPROVED** [review]
LGTM, I'll clean some things up in a follow up PR.

**Pull request merged**